api/src/test/java/org/pmiops/workbench/access/UserServiceAccessTest.java

```diff
@@ -197,7 +203,61 @@ public void test_updateUserWithRetries_register_then_unregister() {
         .isEqualTo(TierAccessStatus.DISABLED);
   }
 
-  private DbUser registerUser(DbUser user) {
+  @Test
+  public void testUpdateUserWithRetriesForAccessRenewal() {
+    providedWorkbenchConfig.access.enableAccessRenewal = true;
+    providedWorkbenchConfig.access.enableDataUseAgreement = true;
```

Comment:
could you add a test where this is false, and show that we do not expire a user which would have expired if this were true?

Reply:
done